src/components/challenge/Challenge.js

```diff
@@ -28,20 +39,20 @@ class Challenge extends Component {
           title: 'What’s the science say?',
           content: `Regular physical activity is important for increasing energy and focus,
 strengthening bones and muscles, and increasing longevity
-[[1]](https://www.cdc.gov/physicalactivity/basics/pa-health/index.htm#StrengthenBonesMuscle).
+${formatLink(1)}.
```

Comment:
Woah is this some type of ES6 string interpolation? If so, about time JavaScript...